give/src/components/Form/Options/Options.js

```diff
@@ -0,0 +1,172 @@
+// Based on v2
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-form/form-options.html
+// https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/form/options/v2/options/options.html
+
+import { ComponentWrapper } from "../../shared";
+import "./Options.css";
+
+export const Options = ({
+  className = "", // string
+  type, // string
+  title, // string
+  name, // string
+  helpMessage = "", // string
+  id, // string
+  options, // array of objects {text, value, selected, disabled}
+}) => {
+  const types = ["checkbox", "radio", "drop-down", "multi-drop-down"];
+
+  if (!types.includes(type) || options.length === 0) return null;
+
+  const isCheckbox = type === "checkbox";
+  const isRadio = type === "radio";
+  const isSelect = type === "drop-down";
+  const isMultiSelect = type === "multi-drop-down";
+
+  const selectValues = () => {
+    let values = [];
+    options.forEach((option) => {
+      if (option.selected) {
+        values.push(option.value);
+      }
+    });
+    if (isMultiSelect) {
```

Comment:
You can use return early and pop too for this
```
if(!isMultiSelect){
return values.pop();
}
return values;
```

Reply:
I forgot about `.pop()`. I simplified it even more I feel.

https://github.com/CruGlobal/cru-content-designs/pull/8/commits/23075605aba1cf5f5cb0893c8fe53c0f116139df